{"name": "异常用例: 手机号重复", "status": "passed", "attachments": [{"name": "添加失败", "source": "e5b35c69-ea10-43c3-a079-36c32961b752-attachment.png", "type": "image/png"}], "start": 1625239372764, "stop": 1625239384569, "uuid": "94d02564-38af-41b4-84be-81a077dda1b8", "historyId": "f010da8f2642d8bb81b3eb223a28a2e0", "testCaseId": "1f80dde89e1a8c0503ff8e0c3c796fc8", "fullName": "tests.test_add_contact.TestAddContact#test_add_contact_fail", "labels": [{"name": "story", "value": "添加成员失败"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_add_contact"}, {"name": "subSuite", "value": "TestAddContact"}, {"name": "host", "value": "ccyangdeMacBook-Pro.local"}, {"name": "thread", "value": "17833-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_add_contact"}]}